rust/scx_userspace_arena/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "scx_userspace_arena"
+version = "1.0.11"
+edition = "2024"
```

Comment:
@JakeHillion this breaks the build on Ubuntu, similar to https://github.com/sched-ext/scx/commit/b4170148639b7b2ea1669514cfd786328bfd8ad2.

Do we really need 2024 here?